libs/sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -2000,6 +2022,7 @@ private static void printStatement(String sql) {
             statement instanceof GCDanglingArtifacts ||
             statement instanceof CreateFunction ||
             statement instanceof CreateUser ||
+            statement instanceof CreateRole ||
```

Comment:
Can we also implement formatting for this?

Reply:
There is a change in the `SQLFormatter`, without it `test_create_role_statement` fails. Or you meant smth different?

Reply:
Nevermind I read the diff wrong, I thought these were the classes to exclude from formatting instead of include